ground/gcs/src/plugins/config/config.pro

```diff
@@ -47,7 +48,8 @@ HEADERS += calibration.h \
     configmodulewidget.h \
     configosdwidget.h \
     convertmwrate.h \
-    expocurve.h
+    expocurve.h \
+    autotuneshareform.h
```

Comment:
Yeah. Perhaps at some point refactoring each of the sections into separate libs could be a good idea?